components/phpunit_bridge.rst

```diff
@@ -0,0 +1,114 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
```

Comment:
Ha right, although maybe it would be better to keep it to me more consistent:

* The DomCrawler Component
* The EventDispatcher Component
* The Filesystem Component
* ...